localisation/000_wc_military_localization.csv

```diff
@@ -1,17 +1,13 @@
 #CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
 #Military Technologies;;;;;;;;;;;;;;x
-TECH_INFANTRY;Conventional Warfare;;Konventionelle Kriegsf�hrung;;;;;;;;;;;x
-TECH_CAVALRY;Specialized Warfare;;Irregul�re Kriegsf�hrung;;;;;;;;;;;x
-TECH_SKIRMISH;Ley Attunement;;Ley Einstimmung;;;;;;;;;;;x
-TECH_MELEE;Tactics Development;;Taktikentwicklung;;;;;;;;;;;x
+TECH_INFANTRY;Frontline Troops;;;;;;;;;;;;;x
```

Comment:
I may not particularly say that Casters and Archers are "frontline". Maybe `Primary Troops`, `Baseline Troops`, or `Principal Troops`?

Reply:
I like Baseline... call Cavalry Primary Troops if you play nomad... meeeh.